src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -48,6 +48,12 @@ static RuntimeMetrics()
             () => GetGarbageCollectionCounts(),
             description: "Number of garbage collections that have occurred since process start.");
 
+        MeterInstance.CreateObservableUpDownCounter(
+            "process.runtime.dotnet.gc.live_objects.size",
+            () => GC.GetTotalMemory(false),
+            unit: "bytes",
+            description: "Count of bytes currently in use by live objects in the GC heap. Fragmentation and other GC committed memory pools are excluded.");
```

Comment:
"live objects" is inaccurate/misleading per @noahfalk's comment https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/683#discussion_r1018055369.

Reply:
description updated in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/683/commits/48e021b49143b68fa59ad9f95c677b72ee71ca78